config/config.go

```diff
@@ -660,17 +660,20 @@ func InitServer(ctx context.Context, currentServers ServerType) error {
 	viper.SetDefault("OIDC.ClientSecretFile", filepath.Join(configDir, "oidc-client-secret"))
 	viper.SetDefault("Cache.ExportLocation", "/")
 	viper.SetDefault("Registry.RequireKeyChaining", true)
+	viper.SetDefault("Shoveler.TokenLocation", "/etc/pelican/shoveler-token")
```

Comment:
Do we ever expect someone running an origin as non-root to have this setup? I don't think they'll have permission to write the token here if that's the case. Maybe you could put this at `filepath.Join(configDir, "shoveler/shoveler-token")`?